cpp/src/groupby/sort/group_rank_scan.cu

```diff
@@ -277,7 +277,7 @@ std::unique_ptr<column> group_rank_to_percentage(rank_method const method,
   CUDF_EXPECTS(percentage != rank_percentage::NONE, "Percentage cannot be NONE");
   auto ranks = make_fixed_width_column(
     data_type{type_to_id<double>()}, group_labels.size(), mask_state::UNALLOCATED, stream, mr);
-  ranks->set_null_mask(copy_bitmask(rank, stream, mr));
+  ranks->set_null_mask(copy_bitmask(rank, stream, mr), rank.null_count());
```

Comment:
`ranks->mutable_view()` is called below so this may be fixed again?

Reply:
Yes, you are correct. I will move it right before the return.
Actually it appears that the null-count here should always be zero since there is no logic in here that checks for nulls in the input `rank` column. And since the output column is initially created with no null-mask, I think calling `set_null_count(0)` would be more appropriate here.